EIPS/eip-4762.md

```diff
@@ -123,31 +119,21 @@ We define **write events** as follows. Note that when a write takes place, an ac
 When a nonzero-balance-sending `CALL` or `SELFDESTRUCT` with a given sender and recipient takes place, process these write events:
 
 ```
-(sender, 0, BALANCE_LEAF_KEY)
-(recipient, 0, BALANCE_LEAF_KEY)
+(sender, 0, BASIC_DATA_LEAF_KEY)
+(recipient, 0, BASIC_DATA_LEAF_KEY)
 ```
 
 When a contract creation is initialized, process these write events:
 
 ```
-(contract_address, 0, VERSION_LEAF_KEY)
-(contract_address, 0, NONCE_LEAF_KEY)
-```
-
-Only if the value sent with the creation is nonzero, also process:
-
-```
-(contract_address, 0, BALANCE_LEAF_KEY)
+(contract_address, 0, BASIC_DATA_LEAF_KEY)
 ```
 
 When a contract is created, process these write events:
 
 ```
-(contract_address, 0, VERSION_LEAF_KEY)
-(contract_address, 0, NONCE_LEAF_KEY)
-(contract_address, 0, BALANCE_LEAF_KEY)
-(contract_address, 0, CODE_KECCAK_LEAF_KEY)
-(contract_address, 0, CODE_SIZE_LEAF_KEY)
+(contract_address, 0, BASIC_DATA_LEAF_KEY)
```

Comment:
Is this line needed given that L126 does the same write-access and `initialized < created`?

Reply:
```suggestion
```